privacy-policy.md

```diff
@@ -31,7 +31,7 @@ If we merge with another company such that your information will become subject
 We use cookies and similar technologies such as pixels and local storage to recognize you when you return to our Services. We use them in various ways, for example to log you in, remember your preferences (such as default language), evaluate email effectiveness, show relevant ads, and personalize information.
 
 We respect Do Not Track (“DNT”) settings in browsers. If you’re logged out of our Services and have DNT enabled, we will not set cookies. 
-You can’t log into Medium while DNT is enabled.
+Logging in overrides your DNT setting if it is enabled.
```

Comment:
I think overrides is a bit strong, we just ignore. How about something like this: "By logging in you are opting to allow us to ignore the DNT setting, cookies will be used to track your usage in order to provide you a personalized experience."

We should also make sure this copy is consistent with the explanatory text shown on the login form.